apps/docs/src/app/platform/component-docs/platform-forms/platform-form-container/platform-form-container-examples/platform-form-container-complex-example.component.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+    selector: 'fdp-platform-form-container-complex-example',
+    templateUrl: './platform-form-container-complex-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformFormContainerComplexExampleComponent {
+    form: FormGroup;
```

Comment:
`form: FormGroup = new FormGroup({})`

Reply:
changed as suggested and removed the constructors, please check. Thanks!